scrunch/accounts.py

```diff
@@ -50,20 +50,11 @@ def current_account(cls, connection=None):
         act_res = site_root.account
         return cls(act_res)
 
-    def execute(self, script_body):
-        """
-        Will run a system script on this account.
-
-        System scripts do not have a return value. If they execute correctly
-        they'll finish silently. Otherwise an error will raise.
-        """
+    def execute(self, script_body, strict_subvariable_syntax=None):
+        """Will run a system script on this account."""
         # The account execution endpoint is a shoji:view
-        payload = shoji_view_wrapper(script_body)
-        try:
-            self.resource.execute.post(payload)
-        except pycrunch.ClientError as err:
-            resolutions = err.args[2]["resolutions"]
-            raise ScriptExecutionError(err, resolutions)
+        system_script = SystemScript(self.resource)
+        system_script.execute(script_body, strict_subvariable_syntax)
```

Comment:
I see that the original wasn't returning anything, should we return the Request obejct? This would allow the caller to get a hold of the progress URL and track it, otherwise the return is None and it's impossible for the caller to know where to track this progress

Reply:
I did that for the projects because I added an integration test, not for this one. Thanks